lib/ops/resources/gravity/collection.go

```diff
@@ -458,8 +458,8 @@ func (r alertCollection) ToMarshal() interface{} {
 }
 
 // Resources returns the resources collection in the generic format
-func (c alertCollection) Resources() (resources []teleservices.UnknownResource, err error) {
-	for _, item := range c {
+func (r alertCollection) Resources() (resources []teleservices.UnknownResource, err error) {
```

Comment:
lol, I remember renaming these to 'c' some time ago because 'c' stands for "collection"; what does `r` stand for I wonder? :smile: